apps/scan/backend/src/scanners/pdi/state_machine.ts

```diff
@@ -242,10 +240,10 @@ function buildMachine({
 
   function createPollingChildMachine(
     id: string,
-    queryFn: (context: Pick<Context, 'client'>) => Promise<Event>,
+    queryFn: () => Promise<Event>,
```

Comment:
why remove the type here?

Reply:
The query function has access to the singleton scanner client in the outer function scope, so it doesn't need to get it from the context anymore